x-pack/plugins/lists/common/schemas/request/create_list_schema.ts

```diff
@@ -17,8 +18,18 @@ export const createListSchema = t.intersection([
       type,
     })
   ),
-  t.exact(t.partial({ deserializer, id, meta, serializer })),
+  t.exact(
+    t.partial({
+      deserializer, // defaults to undefined if not set during decode
```

Comment:
These comments have been super helpful throughout the lists api fyi - thank you!